src/modules/AddQR/QR.js

```diff
@@ -23,20 +23,15 @@ module.exports = exports = function (options, pixels, oldPixels, callback) {
                 height = oldPixels.shape[1];
             var xe = width - options.size,
                 ye = height - options.size;
+                var pixelSetter = require('../../util/pixelSetter.js');
             for (var m = 0; m < width; m++) {
                 for (var n = 0; n < height; n++) {
                     if (m >= xe && n >= ye) {
-                        pixels.set(m, n, 0, qrPixels.get(m - xe, n - ye, 0))
-                        pixels.set(m, n, 1, qrPixels.get(m - xe, n - ye, 1))
-                        pixels.set(m, n, 2, qrPixels.get(m - xe, n - ye, 2))
-                        pixels.set(m, n, 3, qrPixels.get(m - xe, n - ye, 3))
+                        pixelSetter(m,n,[qrPixels.get(m - xe, n - ye, 0),qrPixels.get(m - xe, n - ye, 1),qrPixels.get(m - xe, n - ye, 2),qrPixels.get(m - xe, n - ye, 3)],pixels);
```

Comment:
This looks really good, however, I wonder if you could break it up over multiple lines to make it a bit more readable? Maybe like:

```suggestion
                        pixelSetter(m, n, [
                            qrPixels.get(m - xe, n - ye, 0),
                            qrPixels.get(m - xe, n - ye, 1),
                            qrPixels.get(m - xe, n - ye, 2),
                            qrPixels.get(m - xe, n - ye, 3)],
                          pixels);
```

Reply:
yeah sure I ll have a look at it, but can we keep this on hold for sometime, actually currently I am trying to figure out another soc task and once I am done with it can we then do this one ?